design.md

```diff
@@ -41,7 +41,7 @@ The kernel runs a gRPC server that supports the following operations.
   - The kernel can ask the IDE for confirmation. Confirmation schema:
     - Description
     - Confirmation response: Allowed/Denied
-  - A progress of `NaN` is an "unsure of progress" progress bar (I forget what this is called).
+  - A progress of `NaN` must show an indeterminate progress bar. Otherwise, progress is a percentage stored as an integer in the range `[0, 100]`.
```

Comment:
The gRPC type would be `float` or `double` [1] in order to support `NaN`, so is there a reason to restrict other values to integers? as I see it, that can be done on the client side if needed by the particular UI library, but shouldn't be a restriction at the protocol level.

[1]: https://developers.google.com/protocol-buffers/docs/proto3#scalar